docs/TEAMS.md

```diff
@@ -0,0 +1,109 @@
+Organic Maps is led by community contributors, with functional domains occasionally establishing or dissolving their leadership structures, formal or informal. 
+
+This file outlines the functional project structure that has evolved over time. The current list of teams is available on [GitHub](https://github.com/orgs/organicmaps/teams), accessible only to [GitHub Org members](https://github.com/orgs/organicmaps/people).
+
+Below is the list of teams and groups to tag on GitHub if you need assistance in a specific area. If you are unsure which team to contact, please tag  [@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage) for help.
+
+## Contributors
+
+[@organicmaps/contributors](https://github.com/orgs/organicmaps/teams/contributors/teams)
+
+Active code contributors who can create branches and manage issues in the main repo. See [subteams](https://github.com/orgs/organicmaps/teams/contributors/teams).
+
+### Triage
+
+[@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage)
+
+A special subteam to triage incoming issues and pull requests. Please tag this group if you're unsure which group to tag.
+
+## Maintainers
```

Comment:
I'd rename it to Mergers for clarity.

Reply:
Yeah, it is really about doing the final check that all conditions are met and pressing the button. The road to merging a PR should be clear.

Reply:
```suggestion
## Mergers
```

Reply:
Applied.